File: Core/Entities/TheatricalEvent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Core.Entities
{
    public class TheatricalEvent
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Duration { get; set; }
        public byte[] Image { get; set; }
        public List<ScheduledEvent> ScheduledEvents { get; set; }
    }
}

File: Clean_Angular/ClientApp/src/app/models/theatricalevent.ts
import { Time } from "@angular/common";
import { ScheduledEvent } from "./scheduledevent";

export class TheatricalEvent {
    constructor(
        public id?: number,
        public name?: string,
        public description?: string,
        public duration?: Time,
        public image?: string,
        public scheduledEvents?: ScheduledEvent[]) { }
}
